dropwizard-jersey/src/main/java/io/dropwizard/jersey/params/NonEmptyStringParamFeature.java

```diff
@@ -0,0 +1,40 @@
+package io.dropwizard.jersey.params;
+
+import com.google.common.base.Optional;
+
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.ParamConverterProvider;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+/**
+ * A class for describing how Jersey should serialize a {@link NonEmptyStringParam}. If the
+ * parameter was not detected in the response, instead of the resulting value being null, it will
+ * evaluate to {@link Optional#absent()}
+ */
+final public class NonEmptyStringParamFeature implements Feature {
+    @Override
+    public boolean configure(final FeatureContext context) {
```

Comment:
Minor nitpic: We don't require to mark all method parameters or classes as final. This applies only to local variables,